### PR TITLE
Adds animal hospital ruin

### PR DESCRIPTION
**Pull request by Cheridan**
zmvnmjt

### CODE REVIEW

**Comment by Bawhoppen** [comment]
CHERIDAN I DON'T CARE IF THIS IS A MEME OR NOT YOU NEED GODDAMN VARIETYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYY

**Comment by xxalpha** [comment]
>non tgm map

:question: :grey_question: :question: :grey_question: :question: :grey_question: :question: :grey_question:

**Comment by Cheridan** [comment]
i'll format it later it's really late :moon:

**Comment by GunHog** [comment]
Pics????

**Comment by Iamgoofball** [comment]
@xxalpha Map loader doesnt support tgm

**Comment by lzimann** [comment]
@Iamgoofball #16969 soon™

**Comment by Cheridan** [comment]
@GunHog if you must truly spoil the sensation of discovery for yourself this is what it will look like in-game http://i.imgur.com/PxL0RhW.png (sans the honk turfs of course, i loaded it as its own z-level for testing)

**Comment by Jalleo** [comment]
Not a custom bloodied washing machine thats broken and non interactible to reference the old corgi washing.

**Comment by optimumtact** [comment]
Thumbs up

File: code/modules/awaymissions/corpse.dm [hunk 1]
```diff
@@ -216,6 +216,25 @@
 	qdel(src)
 
 
+/obj/effect/mob_spawn/mouse
+	name = "sleeper"
```
Comment (KorPhaeron) [inline code/modules/awaymissions/corpse.dm]: should these not have special names or anything? Or is the joke that players spawn as a cow

**Comment by KorPhaeron** [comment]
This is probably gonna end up weird/ignored after a week but here goes

**Comment by lzimann** [comment]
just like all new features

File: code/modules/awaymissions/corpse.dm (hunk 1 above)
Comment (Cheridan) [inline code/modules/awaymissions/corpse.dm]: the names/flavour text are set on the map

**Pull request merged**